### PR TITLE
port to python 38

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545

### CODE REVIEW

**Comment by bissenbay** [comment]
@fridex @harshad16 We don't have `.aicoe-ci.yaml` file in this repo, is it expected?

**Comment by fridex** [comment]
> @fridex @harshad16 We don't have `.aicoe-ci.yaml` file in this repo, is it expected?

I think it will need to be added as suggested by @harshad16 to make tests run on py38.

**Comment by harshad16** [comment]
can you please include the `.aicoe-ci.yaml`
```
check:
  - thoth-build
  - thoth-precommit
  - thoth-pytest-py38
build:
  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
  build-stratergy: Source
  registry: quay.io
  registry-org: thoth-station
  registry-project: adviser
  registry-secret: thoth-station-thoth-pusher-secret
deploy:
  project-org: "thoth-station"
  project-name: "thoth-application"
  image-name: "adviser"
  overlay-contextpath: "adviser/overlays/test/imagestreamtag.yaml"
```

**Comment by bissenbay** [comment]
> can you please include the `.aicoe-ci.yaml`
> 
> ```
> check:
>   - thoth-build
>   - thoth-precommit
>   - thoth-pytest-py38
> build:
>   base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
>   build-stratergy: Source
>   registry: quay.io
>   registry-org: thoth-station
>   registry-project: adviser
>   registry-secret: thoth-station-thoth-pusher-secret
> deploy:
>   project-org: "thoth-station"
>   project-name: "thoth-application"
>   image-name: "adviser"
>   overlay-contextpath: "adviser/overlays/test/imagestreamtag.yaml"
> ```
> 
> and also rebase the pull -request , the pipfile.lock is having conflicts.

thanks @harshad16 , https://github.com/thoth-station/adviser/pull/1590/commits/0bb5808efaa68bf9e31cd5ff6d54d6aea4d0b9ea

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+check:
+  - thoth-build
+  - thoth-precommit
+  - thoth-pytest-py38
+build:
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
```
Comment (harshad16) [inline .aicoe-ci.yaml:6]:
```suggestion
  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
```

**Comment by harshad16** [comment]
@fridex , this is now good to merge 
the pytest issue would be fixed once this pr is in master.

**Comment by fridex** [comment]
> @fridex , this is now good to merge
> the pytest issue would be fixed once this pr is in master.

Awesome, thanks! :+1:

**Pull request merged**